### PR TITLE
add seed in response object

### PR DESCRIPTION
**Pull request by jonathanlei**
add wallet seed back in successful response objects (if the user didn't provide a funding destination address)

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
-  xAddress: string,
-  classicAddress: string,
-  address: string,
-  tag?: number | boolean
+  xAddress: string;
+  classicAddress: string;
+  address: string;
+  tag?: number | boolean;
+}
+export interface ResponseType {
```
Comment (ckniffen) [inline src/types.ts:7]: nit: can you name this something less generic like `AccountResponse` or `FundedResponse`.

**Review by ckniffen — APPROVED** [review]

File: src/routes/accounts.ts [hunk 2]
```diff
@@ -83,11 +84,16 @@ export default async function (req: Request, res: Response) {
     }
 
     const status = result.engine_result;
-    const response = {
-      account,
-      amount: Number(amount),
+
+    const response: FundedResponse = {
+      account: account,
+      amount: amount,
```
Comment (JST5000) [inline src/routes/accounts.ts:90]: Why was this changed from `Number(amount)`?

Comment (jonathanlei) [reply src/routes/accounts.ts]: changed the type to number

**Review by JST5000 — APPROVED** [review]
LGTM
